format/avro/codecs/bytes.go

```diff
@@ -0,0 +1,19 @@
+package codecs
+
+import (
+	"github.com/wader/fq/format/avro/schema"
+	"github.com/wader/fq/pkg/decode"
+)
+
+type BytesCodec struct{}
+
+func (l BytesCodec) Decode(name string, d *decode.D) {
+	// What if its a record with a field called name_len?
+	// using a struct is probably a better idea. But it makes it less usable
```

Comment:
I have grey hairs after thinking how to balance correctness and useability like this :)

Reply:
Do you think it would make sense to consume both the length and string into a single scalar and have the symbolic representation be the string?